api/envoy/api/v2/discovery.proto

```diff
@@ -121,10 +121,10 @@ message DiscoveryResponse {
 //
 // DeltaDiscoveryRequest plays two independent roles. Any DeltaDiscoveryRequest
 // can be either or both of:
-//   1. informing the server of what resources the client has gained/lost interest in,
-//      (resource_names_subscribe and resource_names_unsubscribe)
-//   2. (N)ACKing an earlier resource update from the server
-//      (response_nonce, with presence of error_detail making it a NACK).
+// [1] informing the server of what resources the client has gained/lost interest in,
```

Comment:
Do you know if this will render properly as an enumerated list in RST?

Reply:
Probably not. I just wanted the check to pass. I think the .proto file's appearance is more important than the generated docs.

Reply:
Changed the text to just have a couple of numbered items within a paragraph. Now neither version needs nice list formatting.

Reply:
As a general principle, we care about both, Envoy's quality user facing docs, including its APIs, are one of our distinguishing features.